app/assets/stylesheets/templates/_question.scss

```diff
@@ -24,4 +24,7 @@
       }
     }
   }
+  .question-wrapper {
```

Comment:
Anu wants to limit all intake pages to a max width of 570px instead of a % of the width of the container.

Reply:
We moved this to a variable: `$intake-max-width`.